src/api/pipelineRuns.js

```diff
@@ -170,30 +173,46 @@ export function createPipelineRun({
   return post(uri, payload).then(({ body }) => body);
 }
 
-export function rerunPipelineRun(pipelineRun) {
-  const { annotations, labels, name, namespace } = pipelineRun.metadata;
+export function generateNewPipelineRunPayload(pipelineRun, rerun) {
```

Comment:
```suggestion
export function generateNewPipelineRunPayload({ pipelineRun, rerun }) {
```

This makes it easier to understand the code when you see a call to this function:

`…generateNewPipelineRunPayload({ pipelineRun, rerun: true })`
instead of
`…generateNewPipelineRunPayload(pipelineRun, true)`

Reply:
Modern IDEs are cable of solving such problems without making it more complex (adding 2 `{`, `}` with adding for each function `{`, `}` and `rerun`).
![image](https://user-images.githubusercontent.com/9289172/212803369-846d372c-4f33-4863-b700-864f00b5a689.png)
Github supports definition display and step into (for some languages)

Reply:
There are other benefits beyond the self-documenting nature of the code, e.g. consistency with the rest of the codebase, and easier refactoring if new params added in future (especially with any optional / default values) as order won't affect call-sites.